app/models/department.rb

```diff
@@ -0,0 +1,10 @@
+class Department < ApplicationRecord
+
+  scope :sorted, lambda {order("position ASC")}
+
+  validates :name, :presence => true, :uniqueness => true
```

Comment:
Always follow the new Hash syntax `presence: true` not :presence => true.
Same instruction for rest of the hashes